docs/en/contribute_en.md

```diff
@@ -0,0 +1,33 @@
+# Contribute to MyUoM
+
+> You can also contribute in ways that do not require advanced technical knowledge, if you are not yet familiar with technologies so that when you want to do something more advanced you will have a good overview of MyUoM.
+
+Below is a list of things a person can do that are always useful for the application.
+
+- Report any bugs you encounter, recording them in a gitlab issue
+  or by talking to team members so that they can communicate it to people who can fix it.
+- Translate MyUoM into other languages(e.g. German,French) or fix existing translations. (Consult [translations.md]() for more details)
+- Suggest things you would like to see in the application that you think would be useful to several P.M.A.C. students.
+- Make some implementable design (mobile compatible) that you think would make the app more beautiful and usable.
+- Suggest modifications that relate to the user experience while using the app (e.g. maybe in the menu it would be nice if the options were divided based on that)
+
+> If none of this excites you, you can start by contacting members of the team to see if there is a need not listed above.
+
+Now for those who want to add something to the app having knowledge of React and Chakra-UI(it's component library is quickly learned), it would help them to read [codebase.md](https://gitlab.com/opensourceuom/myUoM/-/blob/main/docs/codebase.md) for a start where there is more detail about the code written.
```

Comment:
```suggestion
Now for those who want to add something to the app having knowledge of React and Chakra-UI(it's component library is quickly learned), it would help them to read [codebase.md](https://github.com/Open-Source-UoM/MyUoM/-/blob/main/docs/codebase.md) for a start where there is more detail about the code written.
```
again github link